scripts/run_seqware_workflow.py

```diff
@@ -98,14 +98,16 @@ def write_ini(args):
     else:
         raise Exception("bbFrom must be a local file or a valid URL")
 
+    # check run_id
+    run_id = args.run_id if args.run_id is not None else ""
```

Comment:
It makes sense if `run_id` only contain `[a-zA-Z0-9_-]`, not sure if CWL can do that or not. Anyway, it would be good to check it in the python code here.